element_lab/lab.py

```diff
@@ -77,18 +77,52 @@ class Protocol(dj.Lookup):
     protocol                : varchar(16)
     ---
     -> ProtocolType
-    protocol_description=''        : varchar(255)
+    protocol_description='' : varchar(255)
     """
 
 
 @schema
 class Project(dj.Lookup):
     definition = """
+    -> Lab
```

Comment:
This design forces project to be under lab, a project must be associated with a lab. I think Project is a separate entity, a number of sessions may be associated with a Project, but no need to tie this to Lab.
For example, sessions from multiple different labs can be a part of one project.

Reply:
Good note, I hadn't thought about the need for a many-to-many relationship there. Thanks!